test/e2e/reconfigure_test.go

```diff
@@ -3,23 +3,51 @@ package e2e
 import (
 	"context"
 	"encoding/json"
+	"fmt"
+	"log"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 
+	"github.com/openshift/windows-machine-config-operator/pkg/metadata"
 	"github.com/openshift/windows-machine-config-operator/pkg/patch"
 	"github.com/openshift/windows-machine-config-operator/pkg/wiparser"
 )
 
 func reconfigurationTestSuite(t *testing.T) {
 	tc, err := NewTestContext()
 	require.NoError(t, err)
+	t.Run("Remove version annotation", tc.testRemoveVersionAnnotation)
```

Comment:
Only thing I am worried about is how much time this may add to our test runs. Consider running this only on 1 or 2 platforms (AWS + something else) or just not on azure since azure is right up against the timeout

Reply:
makes sense, updated to run only in AWS